### PR TITLE
Use contants string instead of string literals as cacerts secrect name

### PR DESCRIPTION
**Pull request by escoffier**
**Please provide a description of this PR:**
When with external Istiod, we want to beable to load cacerts from a remote cluster, the secrest name is a string literals, we'd better use a constants instread.

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

File: security/pkg/pki/ca/ca.go [hunk 1]
```diff
@@ -57,6 +57,8 @@ const (
 	TLSSecretRootCertFile = "ca.crt"
 	// The standard key size to use when generating an RSA private key
 	rsaKeySize = 2048
+	// ExternalCASecret stores the key/cert of self-signed CA from external Istiod.
+	ExternalCASecret = "cacerts"
```
Comment (hzxuzhonghu) [inline security/pkg/pki/ca/ca.go:61]: Nit: the comments is not correct, it can come from plugin cert

**Pull request merged**